Website/DesktopModules/Hotcakes/Core/Controllers/CartController.cs

```diff
@@ -321,6 +321,7 @@ private List<ShippingRateDisplay> GetRates()
         #region Main Cart Actions
 
         // GET: /Cart/
+        [Obsolete("This action is no longer used to load the minicart content.")]
         [NonCacheableResponseFilter]
         public ActionResult Index()
```

Comment:
@ismaelTartola Is this the unused endpoint you were telling me about?

Reply:
Yes. It was used in the past to load the minicart initial information. But after the implementation of the SPA feature there are no call's directed to this action.

Reply:
@ismaelTartola So, you're saying both the module and the skin object are using the same SPA implementation now?

Reply:
Yes.